src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineProgram.java

```diff
@@ -59,6 +60,10 @@ protected GCSOptions getAuthenticatedGCSOptions() {
         return options;
     }
 
+    protected AuthHolder getAuthHolder() {
+        return new AuthHolder("SparkCommandLineProgram", apiKey);
```

Comment:
OK. I was also considering just "GATK4". That's the name that shows on the web browser when you're authenticating the very first time.